backend/tracim_backend/models/context_models.py

```diff
@@ -1189,6 +1212,29 @@ def pdf_available(self) -> bool:
         else:
             return False
 
+    @property
+    def has_jpeg_preview(self) -> bool:
+        """
+        :return: bool about if jpeg version of content is available
+        """
+        if self.revision.depot_file:
+            from tracim_backend.lib.core.content import ContentApi
+            content_api = ContentApi(
+                current_user=self._user,
+                session=self.dbsession,
+                config=self.config,
+                show_deleted=True,
+                show_archived=True,
+                show_active=True,
+                show_temporary=True,
+            )
+            return content_api.has_jpeg_preview(
+                self.revision.revision_id,
+                file_extension=self.revision.file_extension
+            )
+        else:
+            return False
```

Comment:
same remark: code must be written: "short cases first"

Reply:
done.